src/aind_metadata_mapper/smartspim/models.py

```diff
@@ -6,6 +6,18 @@
 from pydantic import Field
 
 from aind_metadata_mapper.core_models import BaseJobSettings
+from enum import Enum
+
+
+class SlimsImmersionMedium(Enum):
+    """Enum for the immersion medium used in SLIMS."""
+
+    DIH2O = "diH2O"
```

Comment:
Is there a way in how we can do something like aind-data-schema-models where we can simply have a CSV? @miketaormina do you think we will upgrade/change the mediums in the future?

Reply:
The enum was to validate the values coming out of SLIMS and map it, but we can do a str check instead

Reply:
This enum is what is in SLIMS right now.  (It puts them together). The response from SLIMS is then mapped to the schema (with the separation of immersion type and RI value). We can ask for SLIMS to be updated